mini_programs/rock_paper_scissors.py

```diff
@@ -31,15 +30,19 @@ def display_result(player_choice, comp_choice):
     
     result(player_choice, comp_choice)
 
+
 # List of play options
 options = ['Rock', 'Paper', 'Scissors']
 
 while True:
-    computer = options[randint(0, 2)] # Pick rock, paper, or scissors randomly for the computer
-    player = input('\nRock, Paper, Scissors?\nChoose one (blank to quit): ')
+    computer = choice(options) # Pick rock, paper, or scissors randomly for the computer
```

Comment:
`random.choice()` would be better to use here, as it picks random item from iterables